src/chrome/content/rules/yplanapp.com.xml

```diff
@@ -1,11 +1,12 @@
 <ruleset name="YPlan">
 
-	<target host="yplanapp.com" />
-	<target host="www.yplanapp.com" />
+	<target host="*.yplanapp.com" />
 
-  <securecookie host="^(www\.)?yplanapp.com$" name=".+" />
+	<securecookie host="^.*\.yplanapp\.com$" name=".+" />
```

Comment:
It [does not](https://regex101.com/r/zG4kM9/1) match the domain without any subdomain (here: `https://123-reg.co.uk`). So if this is the aim of the ruleset it's indeed wrong.